load/loader.py

```diff
@@ -102,7 +102,16 @@ def load_voting_history(opts, conf):
         '-param:reportFileDir={}'.format(opts.input_file),
         '-param:reporterKey={}'.format(opts.reporter_key)
     ])
-
+    elif opts.state == 'ny':
+        fl_path = os.path.dirname(os.path.join(conf['data_path'],
```

Comment:
That sounds good to me! We might be able to generalize that later (we have a similar setup accepting a directory or file in transformer), but for now I think this works